spec/carin_for_blue_button/c4bb_client/c4bb_client_initial_wait_test_spec.rb

```diff
@@ -78,16 +78,16 @@ def run(runnable, inputs = {})
     allow(test).to receive_messages(suite:, parent: suite)
 
     patient_fhir_api_request = stub_request(:get, server_patient_api_request)
-                               .with(
-                                 headers: { 'Authorization' => "Bearer #{access_token}" }
-                               )
-                               .to_return(status: 200, body: c4bb_patient_search_bundle.to_json)
+      .with(
+        headers: { 'Authorization' => "Bearer #{client_id}" }
```

Comment:
In the spirit of tests serving as documentation, it would be better if this was an encoded token containing the client ID rather than the token